src/Hangfire.Tags/Storage/ITagsStorage.cs

```diff
@@ -7,7 +7,7 @@ namespace Hangfire.Tags.Storage
     /// <summary>
     /// Abstraction over Hangfire's storage API
     /// </summary>
-    internal interface ITagsStorage : IDisposable
+    public interface ITagsStorage : IDisposable
```

Comment:
Not required for the connection leak issue

Reply:
yup, absolutely, it should be a different PR.